components/Tabs.js

```diff
@@ -1,6 +1,6 @@
 import Link from "@components/Link";
+import Select from "@components/form/Select"
```

Comment:
Please run the linter, missing semi colon to match our project coding standards

```suggestion
import Select from "@components/form/Select";
```